go.work.sum

```diff
@@ -0,0 +1,10 @@
+cloud.google.com/go/compute/metadata v0.2.0/go.mod h1:zFmK7XCadkQkj6TtorcaGlCW1hT1fIilQDwofLpJ20k=
```

Comment:
Not sure what this file is, so just double-check if it's supposed to be here or not (maybe 1.18?)

Reply:
This file pinned some module's dependencies declared in the module's go.mod files, more here: https://go.googlesource.com/proposal/+/master/design/45713-workspace.md#files This file was autogenerated when building the modules after @laurentluce added the Go [workspace feature here](https://github.com/kurtosis-tech/kurtosis/pull/596) and I'm adding the file in the VCS because we do same for mdoule's go.sum files, but we could ignore it if you think that it's the best.

Reply:
if you @mieubrisse think that we should ignore it, let me know and I'll do the change in a next PR